src/routes/game/components/dialogs/components/RulesDialog.vue

```diff
@@ -1,217 +1,50 @@
-          hero and win immediately.
-        </p>
-        <div>
-          <ol>
-            <li>No Kings: 21 points to win (default)</li>
-            <li>1 King: 14 points to win</li>
-            <li>2 Kings: 10 points to win</li>
-            <li>3 Kings: 5 points to win</li>
-            <li>4 Kings: 0 points to win</li>
-          </ol>
-        </div>
-      </section>
-      <section>
-        <h2>FAQ</h2>
-        <h4>
-          Can I play a two to counter a point card? What about a scuttle? Can I play a two to destroy my
-          opponent's Face Card or Glasses Eight on their turn?
-        </h4>
-        <p>
-          Nope! Twos can only counter One-Offs. You can play a two to destroy a Face Card or Glasses Eight,
-          but this takes your turn, unlike countering.
-        </p>
-
-        <h4>Do Queens protect against countering with a two?</h4>
-        <p>
-          Yes! Queens protect against targeting, when a card applies a unique effect to one specific card.
-          This means your other cards (not the Queen, herself) are protected against Twos (either effect),
-          Nines, and Jacks.
-        </p>
-
-        <h4>Can I counter a Two with a Two?</h4>
-        <p>
-          Heck yeah! Playing a Two to counter is a One-Off, which can be countered with a two. Stacked
-          counters make for exciting plays.
-        </p>
-
-        <h4>Can I win by playing a King?</h4>
-        <p>
-          Yes. If you meet the required number of points, you win immediately. This means if you play a King
-          when you already have enough points to meet the new limit, you win on that turn.
-        </p>
-
-        <h4>If I play an Ace or a Six, are my cards destroyed as well?</h4>
-        <p>
-          Yes. Aces and Sixes destroy all Point Cards and all Face Cards plus Glasses Eights, respectively.
-          That includes any that you have out. Try to avoid destroying many of your own cards!
-        </p>
-      </section>
+      <RulesView :is-in-modal="true" :parent-modal-id="'#rulesDialog'" />
     </template>
     <template #actions>
-      <v-btn variant="outlined" color="surface-1" @click="close">
+      <v-btn variant="outlined" color="surface-1" @click="emit('close')">
         Got It!
       </v-btn>
     </template>
   </BaseDialog>
 </template>
 
-<script>
-import BaseDialog from '@/components/BaseDialog.vue';
+<script setup>
+  import BaseDialog from '@/components/BaseDialog.vue';
+  import RulesView from '@/routes/rules/RulesView.vue';
+
+  const emit = defineEmits(['close']);
 
-export default {
-  name: 'RulesDialog',
-  components: {
-    BaseDialog,
-  },
-  emits: ['close'],
-  computed: {
-    buttonSize() {
-      return this.$vuetify.display.mdAndDown ? 'small' : 'medium';
-    },
-  },
-  methods: {
-    close() {
-      this.$emit('close');
-    },
-  },
-};
 </script>
+
 <style lang="scss" scoped>
 .rules-button {
   width: 100%;
 }
+.closeBtn {
+  right: 30px;
+  z-index: 19;
+  top: 15px;
```

Comment:
Prefer multiples of 4
```suggestion
  top: 16px;
```